### PR TITLE
README.md

### PR DESCRIPTION
**Pull request by userNotAvailableE**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,2 @@
 # comeback-java
-only java java java... 
-
```
Comment (cyrusjetson) [inline README.md:3]: y  removed all???

Comment (cyrusjetson) [reply README.md:3]: ???

Comment (userNotAvailableE) [reply README.md:3]: ib donno what i can put jhere

Comment (cyrusjetson) [reply README.md:3]: java is the medicine

**Review by cyrusjetson — APPROVED** [review]

**Pull request merged**